conans/test/unittests/util/tools_test.py

```diff
@@ -729,6 +814,85 @@ def get_file():
 
         thread.stop()
 
+    @attr('slow')
+    @attr('local_bottle')
+    def test_get_mirror(self):
```

Comment:
These tests are complex and low. Given that the ``get()`` is just using ``download()``, couldn't we use the same test to check the 2 things?

Reply:
done, I've merged the tools.download test to tools.get. I thought was better keeping separated, but as we review any new change I can say it's impossible having an extra step in tools.get which affects only its behavior and keeps tools.download, which would create a half-covered case.